src/main/java/rtg/world/gen/ChunkProviderRTG.java

```diff
@@ -291,22 +291,28 @@ public Chunk provideChunk(final int cx, final int cz) {
                 baseBiomesList[k] = biomePatcher.getPatchedBaseBiome("" + Biome.getIdForBiome(landscape.biome[k].baseBiome));
             }
         }
-        volcanoGenerator.generateMapGen(primer, worldSeed, worldObj, cmr, mapRand, cx, cz, rtgWorld.simplex, rtgWorld.cell, landscape.noise, landscape.biome[0]);
+        volcanoGenerator.generateMapGen(primer, worldSeed, worldObj, cmr, mapRand, cx, cz, rtgWorld.simplex, rtgWorld.cell, landscape.noise);
         TimeTracker.manager.stop(volcanos);
         
         String replace = "RTG Replace";
         TimeTracker.manager.start(replace);
         replaceBlocksForBiome(cx, cz, primer, landscape.biome, baseBiomesList, landscape.noise);
         TimeTracker.manager.stop(replace);
 
-        caveGenerator.generate(worldObj, cx, cz, primer);
-        ravineGenerator.generate(worldObj, cx, cz, primer);
+        if (Biome.getIdForBiome(baseBiomesList[0])!=0
+                &&Biome.getIdForBiome(baseBiomesList[0])!=32) {
```

Comment:
Having an hardcoded check here for biome ID 0 and 32 is unwise. What purpose does it serve?